.storybook/storyshots-test.js

```diff
@@ -36,13 +36,13 @@ const beforeScreenshot = () =>
 // Allow a small amount of deviation to account for
 // CI running on a different OS than local development.
 const getMatchOptions = () => ({
-  failureThreshold: 0.1,
+  failureThreshold: 0,
```

Comment:
Changing this so we can actually capture deviences in visual changes - not sure on its affect on CI